src/main/scala-2.11/markets/engines/CDAMatchingEngine.scala

```diff
@@ -137,13 +137,6 @@ class CDAMatchingEngine(askOrdering: PriceOrdering[AskOrder],
   /* Mutable collection of bid orders for internal use only! */
   protected val _bidOrderBook = mutable.TreeSet.empty[BidOrder](bidOrdering)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Field name does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1843449429/issues/source?bid=2921361&fileBranchId=3082852#l138)